src/applications/disability-benefits/all-claims/utils.jsx

```diff
@@ -501,3 +504,28 @@ export const showPtsdAssaultConclusion = form =>
   form['view:upload781aChoice'] === 'answerQuestions' &&
   (_.get('view:selectablePtsdTypes.view:mstPtsdType', form, false) ||
     _.get('view:selectablePtsdTypes.view:assaultPtsdType', form, false));
+
+export const ancillaryFormUploadUi = (label, itemDescription) =>
```

Comment:
Should there be a default item description, in case none is supplied as is done above?

Reply:
I think in this case that wouldn't be necessary since the description itself is approved by DSVA and we wouldn't want a default actually displayed anywhere.